events/class_models/event_models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+
+
+class Event(models.Model):
+    class Colors(models.TextChoices):
+        RED = "#FF0000", "Red"
+        PINK = "#FFC0CB", "Pink"
+        ORANGE = "#FFA500", "Orange"
+        YELLOW = "#FFFF00", "Yellow"
+        PURPLE = "#800080", "Purple"
+        GREEN = "#008000", "Green"
+        BLUE = "#0000FF", "Blue"
+        BROWN = "#A52A2A", "Brown"
+        WHITE = "#FFFFFF", "White"
+        GRAY = "#808080", "Gray"
+        BLACK = "#000000", "Black"
```

Comment:
Because we are going to reference this class in the view then I would suggest putting this class outside of the Event class.

Reply:
Done